src/types/index.ts

```diff
@@ -464,47 +448,53 @@ export interface EnableGeneralTransferManagerProcedureArgs {
 }
 
 /**
- * Arguments for [[EnableCountTransferManagerProcedure]]
+ * Arguments for the [[EnableCountTransferManager]] Procedure
  */
 export interface EnableCountTransferManagerProcedureArgs {
   /**
    * symbol of the Security Token
    */
   symbol: string;
   /**
-   * maximum number of token holders
+   * maximum number of simultaneous token holders
    */
   maxHolderCount: number;
 }
 
 /**
- * Arguments for [[EnablePercentageTransferManagerProcedure]]
+ * Arguments for the [[EnablePercentageTransferManager]] Procedure
  */
 export interface EnablePercentageTransferManagerProcedureArgs {
   /**
    * symbol of the Security Token
    */
   symbol: string;
+  /**
+   * maximum percentage of the total supply a single token holder can hold
+   */
   maxHolderPercentage: BigNumber;
+  /**
+   * whether to allow primary issuance or not
```

Comment:
This one should be the same as the other instances of `allowPrimaryIssuance` in this file